democracy/models/base.py

```diff
@@ -121,9 +121,16 @@ class Commentable(models.Model):
 
     def recache_n_comments(self):
         new_n_comments = self.comments.count()
+        # if commentable has sections, include them in the total comment count
+        if hasattr(self, 'sections'):
+            for section in self.sections.all():
```

Comment:
Use Django's `Sum` aggregate? (Or at the very least, use `.values_list()` to query only the `n_comments` field you need.